velox/connectors/hive/HiveConfig.h

```diff
@@ -110,6 +110,9 @@ class HiveConfig {
   static constexpr const char* kFileColumnNamesReadAsLowerCaseSession =
       "file_column_names_read_as_lower_case";
 
+  static constexpr const char* kPartitionPathAsLowerCaseSession =
```

Comment:
Which config property in Spark does this correspond to?

The description of Spark PR https://github.com/apache/spark/pull/31669 suggests that there is some broad config setting that controls whether something is treated as case sensitive or not. It is not clear what that something is.

Also, please, update PR description to make it more accurate. Currently, it sounds like Spark always treats partition keys as case sensitive which doesn't seem to be the case.

Reply:
@mbasmanova Yes, you are correct. We can directly utilize the `fileColumnNamesReadAsLowerCase `parameter to control whether the partition path should be converted to lowercase. There is no need to introduce an additional parameter. I have made the necessary updates. And also update the PR description. Please review it again. Thank you.